src/main/webapp/style.css

```diff
@@ -7,6 +7,7 @@
   --failed: #dc4f4f;
   --passed: #4fdc7f;
   --primary: white;
+  --button-size: .3125rem;
```

Comment:
Please put this in it's alphabetical place before merging

Reply:
Done. Thank you!